lib/stat_tracker.rb

```diff
@@ -169,9 +168,25 @@ def worst_coach(season_id)
          lose_percentage 
       end
       worst_coach[0]
-      #require 'pry' ; binding.pry
    end
 
+   def most_accurate_team(season)
+      most_accurate_team = team_accuracy.max_by { |game_team, accuracy| accuracy }.first
+      most_accurate_team_by_season = season_games(season).filter_map do |game|
```

Comment:
I know we discussed the filter_map method together. It would be beneficial to share this with the rest of the team. This was a cool method that you used here.